modules/verifying-serverless-app-deployment.adoc

```diff
@@ -24,3 +31,29 @@ helloworld-go   http://helloworld-go.default.example.com   helloworld-go-4wsd2
 $ curl http://helloworld-go.default.example.com
 Hello World: Go Sample v1!
 ----
++
+----
+$ curl https://helloworld-go.default.example.com
+Hello World: Go Sample v1!
+----
+
+== Self signed certificates
+
+If you receive an error relating to a self signed certificate in the certificate chain, you can add the ``--insecure` flag to the curl command to ignore the error.
+
+.Example
+----
+$ curl http://helloworld-go.default.example.com --insecure
+Hello World: Go Sample v1!
+----
+
+== Certificate authority signed certificates
+
+If your {product-title} cluster is configured with a certificate that is signed by a certificate authority (CA), you must specify this with the curl command.
```

Comment:
This is only the case if the CA is not yet configured with the system (which many of the big CAs are out of the box).

Maybe something like:

> If your {product-title} cluster is configured with a certificate that is signed by a certificate authority (CA) that is not yet globally configured in your system, you must specify this with the curl command.